evutil_rand.c

```diff
@@ -193,9 +193,11 @@ evutil_secure_rng_get_bytes(void *buf, size_t n)
 void
 evutil_secure_rng_add_bytes(const char *buf, size_t n)
 {
-#if !defined(EVENT__HAVE_ARC4RANDOM) || defined(EVENT__HAVE_ARC4RANDOM_ADDRANDOM)
```

Comment:
If you're removing the use of `EVENT__HAVE_ARC4RANDOM_ADDRANDOM`, then why is the check for `arc4random_addrandom` being retained in both build systems?

Reply:
You mean to do like that: c7adfa59d0d4d9b6c70aa6fcf010c1940417621a
Right?